crates/turbopack-ecmascript-runtime/js/src/dev/runtime/base/runtime-base.ts

```diff
@@ -823,12 +830,22 @@ function applyInternal(
     disposedModules
   );
 
+  let error: any;
+  var reportError = function (err: any) {
```

Comment:
Nit: (maybe we should start linting these files on top of TS?)

```suggestion
  const reportError = (err: any) => {
```

Reply:
They should actually be linted already, not sure why eslint is not complaining

Reply:
nvm it was not linted, we didn't have `no-var` (which is broken because it includes `declare var`)